cli/proc_state.rs

```diff
@@ -41,12 +41,16 @@ use std::collections::HashSet;
 use std::env;
 use std::fs::File;
 use std::io::BufReader;
+use std::ops::Deref;
 use std::sync::Arc;
 
 /// This structure represents state of single "deno" program.
 ///
 /// It is shared by all created workers (thus V8 isolates).
-pub struct ProgramState {
+#[derive(Clone)]
+pub struct ProcState(Arc<Inner>);
```

Comment:
Go with `ProcessState`? Saving three characters is not worth the abbrevation

Reply:
Why isn't it? Matches what we do in Deploy

Reply:
Okay, if you prefer it...